lib/api.js

```diff
@@ -18,6 +18,7 @@ function PlexAPI(options, deprecatedPort) {
 
     this.hostname = hostname;
     this.port = deprecatedPort || opts.port || PLEX_SERVER_PORT;
+    this.https = opts.https || false;
```

Comment:
Yep, that makes more sense! I'll add another commit. I might break out that logic in to a couple lines because it's becoming a little hard to read for a ternary.